### ISSUE TITLE
properties-panel-extension does not work with bpmn-js-properties-panel v1.0.0-alpha.3

### ISSUE THREAD

**Issue by Axelcouty** [bpmn-io/bpmn-js-examples#155]
__Describe the Bug__

Hello,

The properties-panel-extension example does not work out of the box:
1. `npm build` shows the following warning
    ```
     WARNING in ./node_modules/bpmn-js-properties-panel/dist/index.esm.js 3253:6-13
    export 'isAny' (imported as 'isAny$1') was not found in 'bpmn-js/lib/util/ModelUtil' (possible exports: getBusinessObject, is)
     @ ./app/index.js 3:0-99 15:22-47 15:49-77
    ```
2. When going on the web page and after click on "create diagram" an error is shown
![image](https://user-images.githubusercontent.com/59728940/153375576-cfadb27b-03d8-4de6-925e-c0ec4ac437f5.png)

__Steps to Reproduce__

1. `git clone git@github.com:bpmn-io/bpmn-js-examples.git`
2. `cd bpmn-js-examples/properties-panel-extension`
3. `npm install`
    Here you can observe the warning
4. `npm start`
5. Go to http://localhost:5000 and click "create a diagram"

__Expected Behavior__

When click on "create a diagram" the diagram modeler is shown instead of an error

__Environment__

- Chrome 89.0.4389.114
- Node 15.5.1
- OS: Debian 10.9
- Library version: [e.g. 2.0.0]

__Additional info__

Changing [this line](https://github.com/bpmn-io/bpmn-js-examples/blob/master/properties-panel-extension/package.json#L51) from
from
```
"bpmn-js-properties-panel": "^1.0.0-alpha.0",
```
to 
```
"bpmn-js-properties-panel": "1.0.0-alpha.0",
```

then doing a clean / npm install does avoid the issue

**Comment by nikku** [comment]
Should get fixed via https://github.com/bpmn-io/bpmn-js-properties-panel/pull/581.

Thanks for reporting.

**Comment by nikku** [comment]
Should get fixed via https://github.com/bpmn-io/bpmn-js-properties-panel/pull/581.

Thanks for reporting.

**Comment by MageeLin** [comment]
> Should get fixed via [bpmn-io/bpmn-js-properties-panel#581](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/581).
> 
> Thanks for reporting.

Nikku, Any plan to publish a new version with this fix?

**Comment by marstamm** [comment]
Hi @MageeLin ,

new version was just published as [1.0.0-alpha.4](https://www.npmjs.com/package/bpmn-js-properties-panel/v/1.0.0-alpha.4)
